index.html

```diff
@@ -64,12 +64,12 @@ <h2>work</h2>
     </section>
     <section class="about">
       <h2>about</h2>
-      <section class="about-content">
-        <section class="about-paragraph">
```

Comment:
Renamed sections because instead of using `.about-content` you can use `.about .content` to target children of `about` that have `content` to clean up the code.